### ISSUE TITLE
Highlighted search results hard to see in dark mode

### ISSUE THREAD

**Issue by henryjw** [thepracticaldev/dev.to#9089]
**Describe the bug**

When performing a search, the highlighted text is hard to see in dark mode.

**To Reproduce**

0. Set theme to dark
1. Go to the home page
2. Click on the search bar and perform a search
3. Try to read the highlighted text
**Expected behavior**
I expect to be able to easily read the highlighted text like I can when using the light theme.

**Screenshots**
![highlighting in dark mode](https://user-images.githubusercontent.com/3443648/86416943-86742c00-bc99-11ea-92d1-2329efa96721.png)

**Desktop (please complete the following information):**

- OS: Windows 10
- Browser: Microsoft Edge (Chromium version)
- Version:  83.0.478.54

**Smartphone (please complete the following information):**

- Device:
- OS:
- Browser:
- Version:

**Additional context**

**Comment by jacobherrington** [comment]
Thanks for the issue! We'll take your request into consideration and follow up if we decide to tackle this issue.

To our amazing contributors: issues labeled `type: bug` are always up for grabs, but for feature requests, please wait until we add a `ready for dev` before starting to work on it.

To claim an issue to work on, please leave a comment. If you've claimed the issue and need help, please ping @thepracticaldev/oss and we will follow up within 3 business days.

For full info on how to contribute, please check out our [contributors guide](https://github.com/thepracticaldev/dev.to/blob/master/CONTRIBUTING.md).

**Comment by jacobherrington** [comment]
I'd love some input from @thepracticaldev/design for color choices here - I definitely think this is an accessibility issue that we should tackle soon.

Anyone looking to make a PR for this might find the `::selection` CSS pseudo-element helpful.

![image](https://user-images.githubusercontent.com/11466782/86417702-6a6d8c00-bc93-11ea-86fb-b1d05577d610.png)
*Lack of iOS Safari support could be an issue*

https://css-tricks.com/overriding-the-default-text-selection-color-with-css/

**Comment by rhymes** [comment]
@jacobherrington as this is a duplicate of the recently opened https://github.com/thepracticaldev/dev.to/issues/9074 - would you mind moving your comment there and closing this one? thanks :)

**Comment by ludwiczakpawel** [comment]
@jacobherrington just FYI, the `::selection` is for cases when you actually select piece of text with cursor (to copy it for example).. in this issue, highlighted element isn't really a `::selection` - it's just a normal inline HTML element with extra styling.
